tests/functional/network_policy/test_network_policy.py

```
@@ -0,0 +1,453 @@
+        """
+        namespace = config.ENV_DATA["cluster_namespace"]
+        sm_obj = OCP(kind="ServiceMonitor", namespace=namespace)
+        service_monitors = sm_obj.get().get("items", [])
+
+        expected_monitors = [
+            "rook-ceph-mgr",
+            "s3-service-monitor",
+        ]
+        sm_names = [sm["metadata"]["name"] for sm in service_monitors]
+        logger.info(
+            f"Found {len(service_monitors)} ServiceMonitors: {sm_names}"
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)